apps/judicial-system/backend/src/app/messages/courtUpload.ts

```diff
@@ -12,14 +12,14 @@ export const courtUpload = defineMessages({
     description: 'Notaður sem nafn á skjalaskrá í Auði.',
   },
   courtRecord: {
-    id: 'judicial.system.backend:court_upload.court_record',
-    defaultMessage: 'Þingbók {courtCaseNumber}',
+    id: 'judicial.system.backend:court_upload.court_record_v2',
+    defaultMessage: 'Þingbók {courtCaseNumber}{date}',
```

Comment:
Why do we not have a space here and skip it in [this line](https://github.com/island-is/island.is/pull/10182/files#diff-267297348ad8b9d18e4497b2f05ceb8641595b17d28c39e63c1617db2e914774R163)?

Reply:
Good point! Will change